common/djangoapps/student/tests/test_models.py

```diff
@@ -8,7 +8,7 @@
 from crum import set_current_request
 from django.contrib.auth.models import AnonymousUser, User  # lint-amnesty, pylint: disable=imported-auth-user
 from django.core.cache import cache
-from django.db.models import signals
+from django.db.models import signals  # pylint: disable=unused-import
```

Comment:
Why is this one needed? Does it register something?

Reply:
There is an Issue [BOM-2408](https://openedx.atlassian.net/browse/BOM-2408) in progress right now which will remove such import statements after testing. I didn't remove these in my PR so as to avoid any failure if possible.